BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -2831,5 +2901,30 @@ void DetachExhaustPrefabs()
             exhaustPrefabs.Clear();
         }
         #endregion
+        public double GetDeltaV()
+        {
+            double specificImpulse;
+            double deltaV;
+            double massFlowRate;
+
+            massFlowRate = (boostTime == 0) ? 0 : boosterMass / boostTime;
+            specificImpulse = (massFlowRate == 0) ? 0 : thrust / (massFlowRate * 9.81);
+            deltaV = specificImpulse * 9.81 * Math.Log(part.mass / (part.mass - boosterMass));
+
+            double mass = part.mass;
+            massFlowRate = (cruiseTime == 0) ? 0 : sustainerMass / cruiseTime;
+            if (boosterMass > 0) mass -= boosterMass;
+            specificImpulse = (massFlowRate == 0) ? 0 : cruiseThrust / (massFlowRate * 9.81);
+            deltaV += (specificImpulse * 9.81 * Math.Log(mass / (mass - sustainerMass)));
+
+            return deltaV;
+        }
+
+        public float DefaultMass()
+        {
+            float mass = part.mass;
+            mass *= 0.10f;
+            return mass;
```

Comment:
Unless you're also doing a modification of initial part mass (in this case part.mass - part.mass/10) this is going to result in missiles gaining mass, which may impart flight performance.